types/natsu.d.ts

```diff
@@ -2,3 +2,15 @@ export type NatsPortReq<T> = {
   headers: any;
   body: T;
 };
+
+export interface NatsInfo {
+  subject: string;
+  request: unknown;
+  response: unknown;
+}
+
+export interface IGetCareProviders extends NatsInfo {
```

Comment:
Will it be better if we turn NatsInfo to generic? 

`type NatsInfo<S, Rq, Rs> = {}`

and the implementation should be
`type GetCareProviders = NatsInfo<"api.v2..., { ids: string[]}, {id: string; name: string}[]>`